paegan-transport/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+    name: paegan-transport
+    version: "1.0.3dev"
+
+source:
+  git_url: https://github.com/axiom-data-science/paegan-transport.git
+  git_tag: db8f957f5f379def417f93c1941d3f20dbbf16bb
+
+build:
+    number: 0
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+        - paegan
+        - fiona
+        - requests
+        - redis-py  # [not win]
+        - geojson
+        - pytables
+
+test:
+    imports:
+        - paegan.transport
+        - paegan.transport.bathymetry
+        - paegan.transport.controllers # [not win]
```

Comment:
Just out of curiosity.  This is due to the lack of `redis-py` or something else?